adal/src/telemetry/java/com/microsoft/aad/adal/EventStrings.java

```diff
@@ -152,7 +152,7 @@
 
     // Parameter values
     static final String AUTHORITY_TYPE_ADFS = "adfs";
-    static final String AUTHORITY_TYPE_AAD = EVENT_PREFIX + "aad";
+    static final String AUTHORITY_TYPE_AAD = "aad";
 
     static final String AUTHORITY_VALIDATION_SUCCESS = EVENT_PREFIX + "authority_validation_status_success";
```

Comment:
>EVENT_PREFIX [](start = 55, length = 12)

Looks like in ios there are no prefixes for all parameter values. Should we remove them for static final String AUTHORITY_VALIDATION_SUCCESS = EVENT_PREFIX + "authority_validation_status_success"; as well?

Reply:
@GTValentine Yep - can do

Reply:
Corrected in e36273d8a7c69271b1f8dd07d06135aea255f82e

Reply:
Are you sure it's okay to just change it though? What about all those existing queries?

Reply:
While this _is_ reported by our telemetry, there is currently no dashboard that I'm aware of that is using these values.

Digging around in the iOS code, it looks like they have an entirely different value specified to track the status: iOS is using 'yes' and 'no' it appears. This change may be inconsequential from an x-plat perspective unless these things are brought into alignment. I'll post a message into our telemetry channel to see how we want to address this

Reply:
Per sync with @oldalton - let's update these values to `yes` / `no` to make Android match iOS and not force them to make changes too

Reply:
Updated in 1fdf9a2e5ed6d98e98f4976979a843381f3ea734